runtime/jvmti/jvmtiHook.c

```diff
@@ -239,15 +221,23 @@ jvmtiHookThreadDestroy(J9HookInterface** hook, UDATA eventNum, void* eventData,
 	/* The final thread in the system is destroyed very late, JVMTI could be shut down by then */
 
 	if (vmThread->javaVM->jvmtiData != NULL) {
-		/* Deallocate the thread data block for this environment/thread pair, if it exists */
+		j9object_t threadObject = vmThread->threadObject;
+#if JAVA_SPEC_VERSION >= 19
+		void *tlsArray = J9OBJECT_ADDRESS_LOAD(vmThread, threadObject, vmThread->javaVM->tlsOffset);
+		if (NULL != tlsArray) {
+#endif /* JAVA_SPEC_VERSION >= 19 */
```

Comment:
The corresponding closing bracket `}` won't require a `#if/#endif` block.
```suggestion
#if JAVA_SPEC_VERSION >= 19
		void *tlsArray = J9OBJECT_ADDRESS_LOAD(vmThread, threadObject, vmThread->javaVM->tlsOffset);
		if (NULL != tlsArray)
#endif /* JAVA_SPEC_VERSION >= 19 */
		{
```